get-started/index.md

```diff
@@ -111,21 +111,21 @@ Consider this diagram comparing virtual machines to containers:
 
 ![Virtual machine stack example](https://www.docker.com/sites/default/files/VM%402x.png)
 
-Virtual machines run guest operating systems -- note the OS layer in each box.
-This is resource intensive, and the resulting disk image and application state is
-an entanglement of OS settings, system-installed dependencies, OS security
-patches, and other easy-to-lose, hard-to-replicate ephemera.
+Virtual machines run guest operating systems &#8212; note the OS layer in each
```

Comment:
@mstanleyjones , yes those are emdashes to replace the `--` that were in the original